addons/udes_stock/models/stock_picking_type.py

```diff
@@ -70,6 +70,12 @@ class StockPickingType(models.Model):
         help="When True, operations are automatically validated in real time.",
     )
 
+    u_handle_partials = fields.Boolean(
```

Comment:
Can we add this to the picking type view?

Reply:
Added, please review